tamr_unify_client/dataset/resource.py

```diff
@@ -98,7 +99,7 @@ def upsert_from_dataframe(
         Args:
             df: The data to upsert records from.
             primary_key_name: The name of the primary key of the dataset.  Must be a column of `df`.
-            ignore_nan: Whether to convert `NaN` values to `null` before upserting records to Tamr. If `False` and `NaN` is in `df`, this function will fail. Optional, default is `True`.
+            ignore_nan: Legacy parameter that does nothing
```

Comment:
Same question about deprecation/later removal. Should we print a deprecation warning when people use this parameter?